icu4c/source/i18n/unicode/timezone.h

```diff
@@ -317,6 +317,17 @@ class U_I18N_API TimeZone : public UObject {
      */
     static TimeZone* U_EXPORT2 createDefault(void);
 
+    /**
+     * If the locale contains the timezone keyword, creates a copy of that TimeZone.
+     * Otherwise, create the defaul timezone.
```

Comment:
s/defaul/default/

Also, I would rather see this as `@draft ICU 68` than internal.  The methods above and below are stable, so why not this one?

Reply:
then we need to make an API proposal, right? But why do we need a public API for this?

Reply:
I think it would be a useful public API; it makes it easier to use the `u-tz` extension, which many applications are probably currently ignoring.  However, you can check in this PR with it as `@internal` and open a follow-up ticket to promote it to `@draft` later.